crates/uv/tests/it/venv.rs

```diff
@@ -656,13 +656,13 @@ fn create_venv_respects_group_requires_python() -> Result<()> {
 
     uv_snapshot!(context.filters(), context.venv().arg("--python").arg("3.11"), @r"
     success: false
-    exit_code: 1
+    exit_code: 2
```

Comment:
I think these error code changes are fine?

Reply:
FWIW, this did break tox-uv, https://github.com/tox-dev/tox-uv/pull/223.

Reply:
No worries, we released a fix, though internally at my company we have a 7 day delay ingest... that did cause some issues, as we only discovered the issue a few days ago, so pulling in the fix too will take a few days 🤦